kernel/kernel-defaults/src/main/java/io/delta/kernel/defaults/internal/parquet/ParquetSchemaUtils.java

```diff
@@ -385,4 +426,64 @@ private static Optional<Integer> getFieldId(FieldMetadata fieldMetadata, String
               return (int) fieldIdLong;
             });
   }
+
+  /**
+   * Validator for checking that the field ids in the schema. As any schema visitor goes through the
```

Comment:
Incomplete sentence: `Validator for checking that the field ids in the schema ____?`